contrib/vcloud-benchmark.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import logging
+import os
+import sys
+
+sys.dont_write_bytecode = True  # prevent creation of .pyc files
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), ".."))
+
+from vcloud.benchmarkbase import BenchmarkBase  # noqa E402
+from benchexec import __version__  # noqa E402
+import benchexec.benchexec  # noqa E402
+import benchexec.tools  # noqa E402
+
+
+class Benchmark(BenchmarkBase):
```

Comment:
I guess we should use more descriptive names for both classes. There exists already a class `Benchmark` in BenchExec, for example.

How about something with `Vcloud` in the name? Maybe we can also unify both classes given that one is really short (if keeping them separate is not required for making the integration with CPAchecker's scripts easier).

Reply:
I have tried to keep it such that one needs to only copy the folder `vcloud` to the `CPAchecker` repo.
This file is not required there.

On the other hand, we can also have only one file that would pass different parameters to the main.
I have no strong opinions.

Reply:
> I have tried to keep it such that one needs to only copy the folder `vcloud` to the `CPAchecker` repo.

Yes, this is something that we should aim for and other simplifications should be done only if they do not interfere with this goal.

Reply:
> How about something with `Vcloud` in the name? Maybe we can also unify both classes given that one is really short (if keeping them separate is not required for making the integration with CPAchecker's scripts easier).

done